### PR TITLE
Update outdated migration advice from readme

### PR DESCRIPTION
**Pull request by frankh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,6 @@ You can set up a complete cabot stack easily using docker-compose.
 
 - Copy your cabot config to conf/production.conf
```
Comment (dbuxton) [inline README.md]: should this be `production.env`?

Comment (frankh) [inline README.md]: yes, good catch

File: README.md [hunk 2]
```diff
@@ -46,10 +46,6 @@ You can set up a complete cabot stack easily using docker-compose.
...
 
-- Run the migrations
-
-> `docker-compose run --rm web python /manage.py migrate`
-
```
Comment (frankh) [inline README.md]: This isn't needed in the docker-compose instructions as the docker-compose command runs migrations by default

**Comment by dbuxton** [comment]
lgtm

**Pull request merged**